programs/sbf/rust/invoke/src/lib.rs

```diff
@@ -742,7 +742,7 @@ fn process_instruction<'a>(
                 &create_instruction(
                     *invoked_program_id,
                     &[
-                        (accounts[ARGUMENT_INDEX].key, true, false),
+                        (accounts[ARGUMENT_INDEX].key, false, false),
```

Comment:
Isn't this testing a different code path? Shouldn't we maintain a test case for the `true` (is_writable) case?

Reply:
Well, technically the test never needed to pass a writable account, because it is invoking a no-op program anyway. And typically, the most restrictive (readonly) includes the less restrictive (writable) code paths, but not the other way around.

So yes, we could duplicate the test or run it twice with both configurations. Then again, we could do that in lots of tests. What makes this one special?

Reply:
> What makes this one special?

There is nothing special about this one. I just believe we shouldn't decrease test coverage.